code/modules/clothing/neck/_neck.dm

```diff
@@ -170,26 +170,76 @@
 	icon_state = "christmasscarf"
 	item_color = "christmasscarf"
 
-//The three following scarves don't have the scarf subtype
+//The three following scarves don't have the scarf subtype // yogs - they now have the striped scarf subtype
 //This is because Ian can equip anything from that subtype
 //However, these 3 don't have corgi versions of their sprites
-/obj/item/clothing/neck/stripedredscarf
+
+/obj/item/clothing/neck/stripedredscarf //yogs - needs to exist because its mapped in :annoyed:
```

Comment:
You can't remove all these scarftypes and replace them with your yogs variant. That's just more trouble for me down the road.

Dump all your new scarftypes into ``/yogstation/code/modules/clothing/neck/_neck.dm``.